src/waterfall/waterfall.h

```diff
@@ -128,6 +136,7 @@ class Waterfall : public QQuickPaintedItem
     float _mouseColumnConfidence;
     float _mouseColumnDepth;
     float _mouseDepth;
+    bool _containsMouse;
```

Comment:
Alphabetic order

Reply:
I'd rather keep mouse-related variables together. And the rest is also pretty much out of order. I don't mind making a commit to organize that, but seems out of scope for this PR.

Reply:
https://github.com/bluerobotics/ping-viewer/issues/411

We've got a lot TODO already, I think we can move along now that we have notes of everything.